test/OpenTelemetry.Exporter.Jaeger.Tests/OpenTelemetry.Exporter.Jaeger.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>Unit test project for Jaeger Exporter for OpenTelemetry</Description>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp2.1</TargetFrameworks>

```

Comment:
do we need both? product code only compiles for `netstandard2.0;netstandard2.1` so `netcoreapp3.1` doesn't seem to be testing anything new.

Also it may be the cause of the build failure.

Reply:
Removed. @cijothomas had asked for that on #605 but it's not adding a whole lot of value.